nix/default.nix

```diff
@@ -1,6 +1,11 @@
 { system ? builtins.currentSystem
 , crossSystem ? null
 , config ? {}
+, deploymentGlobalsFn ? ## :: Pkgs -> GlobalsOverlayOverDefaults
+    pkgs:
+    if builtins.pathExists ../globals.nix
+    then import ../globals.nix pkgs
+    else builtins.trace "globals.nix missing, please add symlink" {}
```

Comment:
if this is supposed to stop evaluation, you probably want to use `throw` instead of trace

Reply:
@jonringer, this is just code motion -- the original code did `trace`, not `throw`.